### DISCUSSION TITLE
using jwt

### DISCUSSION THREAD

**Discussion by Carmel558** [adonisjs/core#4402] [Help] [answered]
I wanted to use jwt token for authentication.
By following the documentation. I encounter the following error.

"SyntaxError: The requested module '@adonisjs/auth' does not provide an export named 'providers'"

Please ask for your help and how to integrate it into the controller, along with the refresh token

**Comment by RomainLanz** [comment, marked as answer]
=> https://github.com/adonisjs/core/discussions/4400
